inc/class-admin-settings.php

```diff
@@ -123,8 +123,7 @@ public static function output() {
 		global $current_section, $current_tab;
 
 		do_action( 'ang_settings_start' );
-		wp_enqueue_style( 'ang-poppins', 'https://fonts.googleapis.com/css?family=Poppins:400,500,600&display=swap', array(), '20191009' );
-		wp_enqueue_style( 'ang_settings', ANG_PLUGIN_URL . 'assets/css/admin-settings.css', array( 'ang-poppins', 'dashicons' ), filemtime( ANG_PLUGIN_DIR . 'assets/css/admin-settings.css' ) );
+		wp_enqueue_style( 'ang_settings', ANG_PLUGIN_URL . 'assets/css/admin-settings.css', array( 'dashicons' ), filemtime( ANG_PLUGIN_DIR . 'assets/css/admin-settings.css' ) );
```

Comment:
Do we need `dashicons` dependency here?

```suggestion
		wp_enqueue_style( 'ang_settings', ANG_PLUGIN_URL . 'assets/css/admin-settings.css', array(), filemtime( ANG_PLUGIN_DIR . 'assets/css/admin-settings.css' ) );
```